public/index.html

```diff
@@ -2,6 +2,16 @@
 <html lang="en">
 
 <head>
+  <!-- Google tag (gtag.js) -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-VDP0XQW2SZ"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-VDP0XQW2SZ');
```

Comment:
this is weird, but apparently google anayltics tracking Id is not a secret. you may want to put a filter on the google analytics to only allow analytics sent from squirelsnest.io to be allowed.

probably unnecessary protection given our scope though